theme.js

```diff
@@ -1 +1,5 @@
-import './js/blogtini.js'
+import { init } from  './js/blogtini.js'
+
+init(window, {
+  // Where we'll add customizations
```

Comment:
(as if it was another site importing snd using blogtini)

That's where you define the components. Override bligtini's, and add your own that you plan on using in your markdown pages.